doc/rfc/0000-rfc-repository.md

```diff
@@ -0,0 +1,168 @@
+overhead to existing open source contributors who want to contribute RFCs.
+
+The choice to store RFCs as text files allows them to be used as any other
+documentation. Matching the RFC process to how we already contribute code and
+documentation makes the process more accessible. Text files can be written to
+and read by anybody, including people outside the opentitan.org domain.
+
+Making the existance of private RFCs known in the public repo prevents confusion
+from those without access. They are included in a redacted form to provide at
+least some context on what they relate to and why they are private. They are
+assigned RFC numbers so that they can be included proper if they can be made
+public in the future.
+
+### Alternative: Google Docs
+
+We could instead go all-in on Google Docs and keep RFC documents in one
+shared folder. They would be discoverable here, but unlike any of our existing
+documentation. The RFC content would be stored in the Google Docs format and
+would need converting to be moved or read elsewhere.
+
+### Alternative: GitHub issues
+
+We could instead require that all RFC text is stored in the GitHub issues
+themselves. GitHub issues can only be read through the GitHub interface, though
+the content is in Markdown format and could be copied elsewhere. They would
+still only be discoverable through the RFC labels in GitHub issues. Comments on
+GitHub issues do not have the same reviewing features that pull request comments
+do.
+
+### Alternative: RFCs as documentation files in the monorepo
+
+We could instead store our RFCs as documentation files mixed into the monorepo's
+OpenTitan documentation. This would give us the documentation benefits, but RFC
+PRs would get lost amongst the faster moving development PRs. We would also lose
+a seperation of concerns, though we've already lost that for regular development
+by using a monorepo there.
+
+## Prior Art
+
+* The Rust project has an RFC repo: <https://github.com/rust-lang/rfcs>.
+
+    * They use the template, PR, review, accept/reject workflow.
+    * In place of a technical committee, the relevant team will vote in the PR.
+    * Accepted RFCs prompt a "tracking issue" to track its implementation.
+    * RFC markdown files are rendered into [a book][rfc-book] for easy browsing.
+
+* The Internet has almost 10,000 RFCs as files:
+
+    * These predate Git or modern open source workflows, and are made
+      discoverable through HTML indexes.
+    * The original [documentation conventions][rfc3] RFC highlights the
+      importance of open decision making.
+
+As a meta point, this RFC was written using a modified version of Rust's RFC
+template. I think the headings used here make for good prompts to ensure all
+aspects of a decision are considered.
+
+## Unresolved questions
+
+* Should historic RFCs be brought into the repo?
+* How do we keep track of the progress of an RFC's implementation?
```

Comment:
We could have a section in the RFC with a check list of the implementation milestones (using [GH flavored task list extension](https://github.github.com/gfm/#task-list-items-extension-)). This checklist can then be updated with the PR that adds the given implementation milestone.

Reply:
I've now specified to use tracking issues in the RFC which are linked to from the RFC content, but separate so they can be updated without lots of churn on already settled RFCs.